koku/api/dataexport/serializers.py

```diff
@@ -18,6 +18,7 @@ class Meta:
             'start_date',
             'end_date',
             'status',
+            'bucket_name',
```

Comment:
We should also add bucket_name to create_only_fields because we don't plan to let the user change the request.

Reply:
I was wondering about that, but there was nothing in the story about it being immutable, so I left it as it was. Will make the change now.